luwak/src/test/java/uk/co/flax/luwak/presearcher/PresearcherTestBase.java

```diff
@@ -140,4 +146,69 @@ public void testAnyTokenHandling() throws IOException {
 
     }
 
+    static final BytesRef NON_STRING_TERM = new BytesRef(new byte[]{ 60, 8, 0, 0, 0, 9 });
```

Comment:
Would suggest using BytesRefBuilder and NumericUtils.intToPrefixCoded here, makes it less dependent on a raw BytesRef encoding.